docs/getting-started/first-steps.md

```diff
@@ -0,0 +1,7 @@
+# First Steps
+
+TODO: How to get help
```

Comment:
Thanks for the update, I added some docs, they actually work a little different than you think.